lib/info.js

```diff
@@ -108,10 +114,41 @@ module.exports = function getInfo(link, options, callback) {
     } else {
       dashmpd2 = results[1].dashmpd;
     }
-    gotConfig(options, description, config, dashmpd2, callback);
+    gotConfig(options, description, relatedVideos, config, dashmpd2, callback);
   });
 };
 
+/**
+ * @param {String} body
+ */
+function getRelatedVideos(body) {
+  var jsonStr = util.between(body, 'yt.setConfig(', '</script>');
+  var related = [];
+  if (jsonStr) {
+    try {
+      jsonStr = jsonStr.slice(0, jsonStr.indexOf(');'));
+      jsonStr = jsonStr.replace(/'/g,'"');
+      var s = jsonStr.split(',');
+      var config = JSON.parse(jsonStr);
+      if (config && config.RELATED_PLAYER_ARGS && config.RELATED_PLAYER_ARGS.rvs) {
+        var qStrs = config.RELATED_PLAYER_ARGS.rvs.split(',');
+        for (i in qStrs) {
+          var q = querystring.parse(qStrs[i]);
+          related.push({
+            id: q.id,
+            author: q.author,
+            title: q.title,
+            length_seconds: q.length_seconds
+          });
+        }
+      }
+    }
+    catch (err) {
```

Comment:
Put the try/catch statement around the `JSON.parse()` call, since that's the only line that would be throwing an error.